### DISCUSSION TITLE
Mysql int and boolean type convert to string

### DISCUSSION THREAD

**Discussion by mapplechannel** [redpanda-data/connect#2108] [General]
When I connect to MySQL to read data, I find that int type and Boolean type data will be converted into string. Can I keep the original data type output?
[config.md](https://github.com/benthosdev/benthos/files/12585579/config.md)

**Comment by Jeffail** [comment]
Hey @mapplechannel, please stop opening issues for questions, there's a pinned issue explaining why: https://github.com/benthosdev/benthos/issues/2026

**Comment by mihaitodor** [comment]
I shared a workaround and an explanation in #2105.
